opal/core/api.py

```diff
@@ -371,16 +373,3 @@ def post(self, *args, **kwargs):
             episode.set_tag_names(current_tags, None)
         resp = {'ok': 'Got your referral just fine - thanks!'}
         return _build_json_response(resp)
-
-
-class EpisodeListApi(View):
-    """
```

Comment:
@fredkingham Do you think this is not used anywhere (I can completely believe that it's not, just haven't checked myself...)